src/GafferScene/Instancer.cpp

```diff
@@ -1260,6 +1352,7 @@ Instancer::Instancer( const std::string &name )
 	addChild( new StringPlug( "position", Plug::In, "P" ) );
 	addChild( new StringPlug( "orientation", Plug::In ) );
 	addChild( new StringPlug( "scale", Plug::In ) );
+	addChild( new StringPlug( "inactiveIds", Plug::In, "inactiveIds invisibleIds" ) );
```

Comment:
The default value could constitute a breaking change for folks who happen to have these primitive variables already (which they may well do if sourced from a UsdGeomPointInstancer). Should we default to "" and then use `userDefault` metadata to get newly created nodes setup the way we want?